docs/main/Overview/Configuration.md

```diff
@@ -51,6 +51,22 @@ As you'll notice above, some features are dependent on others and therefore auto
 
 Setting an empty array is the same as setting to `None`.
 
+### Blocked Urls<div class="specs"><i>Connection</i><i>Hero</i></div> {#blocked-urls}
+
+Similar to [`Blocked Resource Types`](#blocked-resources), this allows you to block resources
+from being loaded. Instead of focussing on the type of resource we are focussing on the url instead.
+
+Wild cards can be used as well.
+
+Examples:
+
+```
+http://example.com
+example.com
+*.example.com
```

Comment:
I think you might need to escape *,?,etc for this to work with wildcards. It currently does a String.match on the url for each resource. For "findResource", we are converting each to a regex. It might make sense to allow this to operate in a similar way (eg, you can pass in a list of regexes OR strings, and then just convert them).

Just to get this working for now, we might want to just make sure to escape regex characters like ?, *, etc. See here:

https://github.com/ulixee/hero/blob/494415120e279323e79a1d699c14a320e6721b2c/core/lib/Tab.ts#L1035

Reply:
Meaning what, can you perhaps all the different kind of strings / formats / kinds of pattersn we can use now? Perhaps modify the example already for me? so that I can base the tests based on that? Or point me to the correct codebases, but might be easier if you do it. I'm fine either way.

Reply:
Nothing on your side needs to change, unless you think we should modify the parameters to include Regex objects directly (ie, string | Regexp). This will happen transparently. It ensures *.example.com actually runs a regex, where without it, it would try to match an actual *

Reply:
So my examples are spot on? Or do I miss some possibilities here? Or are there mistakes in it?